C/tests/c4DatabaseEncryptionTest.cc

```diff
@@ -46,12 +47,11 @@ TEST_CASE("Database Key Derivation", "[Database][Encryption][C]") {
                                   C4String password,
                                   C4EncryptionAlgorithm alg) = nullptr;
     string expectedKey;
-    C4EncryptionKey key {};
+    C4EncryptionKey key{};
     SECTION("SHA256") {
         c4key_setPasswordFunc = c4key_setPassword;
         expectedKey = "ad3470ce03363552b20a4a70a4aec02cb7439f6202e75b231ab57f2d5e716909";
-    }
-    SECTION("SHA1") {
+    }SECTION("SHA1") {
```

Comment:
Why on earth did it collapse the next SECTION onto the same line as the closing brace?

I'm guessing the formatter spazzed because Catch's macros confuse it.

Reply:
Luckily not a problem with Clang-Format, just seems CLion's formatter has some limitations